server/server.go

```diff
@@ -247,7 +246,6 @@ func (s *Server) getEvent(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 
 	if _, err := w.Write(response); err != nil {
-		s.logger.Errorf("error: writing json []byte response %v", err)
```

Comment:
Why are we removing this one? Shouldn't this only occur if something goes south?

Reply:
Yes but that is not a problem we can fix. It's a problem with the request and it's their fault. Just sending an HTTP error is fine, no reason to log it, because it doesn't signify anything bad on our end is wrong (for example, a broken DB).

Reply:
Wouldn't errors in the request have already been caught when decoding failed? I think if writing a byte array failed it would pretty much have to be the backend's fault. If not, we should change the error sent from `StatusInternalServerError` to `StatusBadRequest`.

Reply:
The only way a write is going to fail is if something is wrong with the underlying writer of http.ResponseWriter, which is not our fault. I'll change the error to StatusBadRequest.